src/components/i18n/en.tsx

```diff
@@ -0,0 +1,10 @@
+export const messages_en = {
+  topbar_exit_button: "Exit Editor",
+  topbar_exit_button_home: "Home",
+  topbar_exit_button_campaigns: "Campaigns",
+  topbar_exit_button_lists: "Lists",
+  topbar_exit_button_controlPanel: "Control Panel",
+  topbar_save_button: "Save",
+  bottombar_exit_button: "Exit and edit later",
+  bottombar_continue_button: "Continue",
```

Comment:
What do you think about continue following the [original guideline](https://github.com/FromDoppler/doppler-webapp/blob/master/src/i18n/README.md#js-language-files):

> In the result object, define all string properties using [ES6 Template Strings](https://developers.google.com/web/updates/2015/01/ES6-Template-Strings), in order to support line breaks.

```suggestion
  topbar_exit_button: "Exit Editor",
  topbar_exit_button_home: "Home",
  topbar_exit_button_campaigns: "Campaigns",
  topbar_exit_button_lists: "Lists",
  topbar_exit_button_controlPanel: "Control Panel",
  topbar_save_button: "Save",
  bottombar_exit_button: "Exit and edit later",
  bottombar_continue_button: "Continue",
```